settings.gradle

```diff
@@ -0,0 +1,12 @@
+include ":api"
+include ":app"
+
+rootProject.children.each { project ->
+    final String groovyName = "${project.name}.gradle"
+    final String kotlinName = "${project.name}.gradle.kts"
+    project.buildFileName = groovyName
+    if (!project.buildFile.isFile()) {
+        project.buildFileName = kotlinName
+    }
+    assert project.buildFile.isFile() : "File named $groovyName or $kotlinName must exist."
```

Comment:
Are you proposing to use more Gradle Kotlin DSL stuff? You're going to have to have that conversation with the greater team?
Or are you just referring to having the gradle build file names start with the name of the project?